server/src/routes/ExerciseRecommendation.ts

```diff
@@ -0,0 +1,61 @@
+import { Request, Response } from 'express';
+import { User, Exercise, updateExercises } from '../db/model';
+
+
+async function getMatchedExercises(req: Request, res: Response) {
+    const id = req.params._id;
+
+    const user = await User.findById(id)
+
+    if (!user) {
+        return res.status(404).json({ error: 'User not found' });
+    }
+   
+    const fitnessLevel = user.settings.fitnessLevel;
+    
+    console.log(fitnessLevel)
+   
+    let exerciseCount: number;
+    if (fitnessLevel === 'none') {

```

Comment:
Create enums of these values and instead use a switch case on them with Object.keys(). Code will be cleaner and more expandable